contracts/bridge/BridgeFee.sol

```diff
@@ -19,16 +19,36 @@ contract BridgeFee {
         receiver = _receiver;
     }
 
-    function _payKLAYFee(uint256 _fee) internal {
-        if(isValidReceiver() == true && _fee > 0) {
-            receiver.transfer(_fee);
+    function _payKLAYFeeAndRefundChange(uint256 _feeLimit) internal returns(uint256) {
+        uint256 fee = feeOfKLAY;
+
+        if(isValidReceiver() == true && fee > 0) {
+            require(_feeLimit >= fee, "insufficient feeLimit");
+
+            receiver.transfer(fee);
```

Comment:
The word `receiver` looks weird to me. If it receives transaction fee, what about using:

- feeReceiver
- bridgeOperator

or something else?

Reply:
`feeReceiver` looks good. I will change.

Reply:
reflected :)